### PR TITLE
fauu.lisons version 0.1.0.22

### PR DESCRIPTION
**Pull request by oxygen-dioxide**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21959)

### CODE REVIEW

File: manifests/f/fauu/lisons/0.1.0.22/fauu.lisons.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: fauu.lisons
+PackageVersion: 0.1.0.22
+PackageLocale: en-US
+Publisher: Piotr Grabowski
+PackageName: Lisons
```
Comment (ItzLevvie) [inline manifests/f/fauu/lisons/0.1.0.22/fauu.lisons.locale.en-US.yaml:8]:
```suggestion
PackageName: Lisons!
```

File: manifests/f/fauu/lisons/0.1.0.22/fauu.lisons.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: fauu.lisons
+PackageVersion: 0.1.0.22
```
Comment (ItzLevvie) [inline manifests/f/fauu/lisons/0.1.0.22/fauu.lisons.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.1.0
```

File: manifests/f/fauu/lisons/0.1.0.22/fauu.lisons.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: fauu.lisons
+PackageVersion: 0.1.0.22
```
Comment (ItzLevvie) [inline manifests/f/fauu/lisons/0.1.0.22/fauu.lisons.yaml:5]:
```suggestion
PackageVersion: 0.1.0
```

File: manifests/f/fauu/lisons/0.1.0.22/fauu.lisons.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: fauu.lisons
+PackageVersion: 0.1.0.22
```
Comment (ItzLevvie) [inline manifests/f/fauu/lisons/0.1.0.22/fauu.lisons.installer.yaml:5]:
```suggestion
PackageVersion: 0.1.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Looks like Control Panel is showing this as `0.1.0` which means you'll have to update the version folder as well.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**